openassessment/data.py

```diff
@@ -333,4 +334,151 @@ def _use_read_replica(self, queryset):
             queryset.using("read_replica")
             if "read_replica" in settings.DATABASES
             else queryset
-        )
\ No newline at end of file
+        )
+
+
+class AggregateOraData(object):
```

Comment:
What do you think of ``OraAggregateData``?  The original tripped me up because it read like an "Or".  Also, it sounds a bit more like an action than a noun as it stands.